sematic/tests/data/test_settings.yaml

```diff
@@ -0,0 +1,8 @@
+active_profile: default
```

Comment:
we should add a `settings_file_schema` field in case we change the structure of this file again in the future. It will help us be able to automatically migrate the file from the old format to the new one.